lib/abstract_format.ml

```diff
@@ -1027,6 +1030,16 @@ and type_of_sf sf : (type_t, err_t) Result.t =
      let%bind high = sf_high |> type_of_sf |> track ~loc:[%here] in
      TyRange {line; low; high} |> return
 
+  (* record type : t(A) = #state{name :: A} *)
+  | Sf.Tuple (4, [Sf.Atom "type";
+                  Sf.Integer line;
+                  Sf.Atom "record";
+                  Sf.List (
+                      Sf.Tuple(3, [Sf.Atom "atom"; Sf.Integer line_field_types; Sf.Atom name])
```

Comment:
`line_field_types` seams to be line of record name. So could you rename `line_field_types` to `line_name`?

Reply:
That's right, I fixed it.

Reply:
thx!